### DISCUSSION TITLE
login failed

### DISCUSSION THREAD

**Discussion by kelebingqishui** [community/community#14032] [Copilot Conversations]
I logged in github Copilot in IDEA, and then there was no response after this page appeared, and I did not log in successfully. please help me ! 
idea version : 2021.3.3
github copilot version : 1.1.15
Installation: idea marketplace

![image](https://user-images.githubusercontent.com/83260995/161241647-cd96211c-b8ea-42f0-9d6d-3aac13d81946.png)

**Comment by upperclassman** [comment]
请问解决了么，我也出现了同样的问题

**Comment by upperclassman** [comment]
昨天用visual stdio试了也是一样的，然后发现是http代理那块有问题
